cluster/manifests/skipper/skipper-redis.yaml

```diff
@@ -0,0 +1,45 @@
+{{ if eq .ConfigItems.skipper_clusterratelimit "true"}}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    application: skipper-ingress-redis
+    version: v4.0.9
+  name: skipper-ingress-redis
+  namespace: kube-system
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      application: skipper-ingress-redis
+  serviceName: skipper-ingress-redis
+  template:
+    metadata:
+      labels:
+        application: skipper-ingress-redis
+        version: v4.0.9
+    spec:
+      containers:
+      - image: registry.opensource.zalan.do/zmon/redis:4.0.9-master-6
+        name: skipper-ingress-redis
+        ports:
+        - containerPort: 6379
+          protocol: TCP
+        readinessProbe:
+          exec:
+            command:
+            - redis-cli
+            - ping
+          failureThreshold: 3
+          initialDelaySeconds: 10
+          periodSeconds: 60
+          successThreshold: 1
+          timeoutSeconds: 1
+        resources:
+          limits:
```

Comment:
Please add requests as well.

Reply:
requests are the same as limits if unspecified or do we break some internal admission hook?